cvmfs/cloud_testing/run.sh

```diff
@@ -177,46 +154,6 @@ get_test_results() {
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 #
 
-
-while getopts "r:b:u:p:e:a:d:m:c:l:" option; do
-  case $option in
-    r)
-      platform_run_script=$OPTARG
-      ;;
-    b)
-      platform_setup_script=$OPTARG
-      ;;
-    u)
-      testee_url=$OPTARG
-      ;;
-    p)
-      platform=$OPTARG
-      ;;
-    e)
-      ec2_config=$OPTARG
-      ;;
-    a)
-      ami_name=$OPTARG
-      ;;
-    d)
-      log_destination=$OPTARG
-      ;;
-    m)
-      username=$OPTARG
-      ;;
-    c)
-      userdata="$OPTARG"
-      ;;
-    l)
-      client_testee_url=$OPTARG
-      ;;
-    ?)
-      shift $(($OPTIND-2))
-      usage "Unrecognized option: $1"
-      ;;
-  esac
-done
```

Comment:
Two things:

* you are relying on environment (`export`-ed) variables here. Which is okay, but by convention I would change their names to all capital with a namespace. For example: `username` becomes `CT_USERNAME` for *CloudTesting_USERNAME* and so on.
* If you do not read any command line parameters anymore, please remove the usage string in this scirpt